docs/algorithms/sig_stfl/sig_stfl.md

```diff
@@ -0,0 +1,27 @@
+
```

Comment:
Good idea to add this file!  Can you add a little bit of markdown formatting to it?

Reply:
Thanks. Done.